### DISCUSSION TITLE
Album organization

### DISCUSSION THREAD

**Discussion by miguelarios** [photoprism/photoprism#1294] [Q&A] [answered]
I wanted to ask about albums management and recommendations. 

I am the type of user that only uses the originals folder and predominantly backs up my camera roll photos and videos from my phone. It is the majority of my library. I organize all these types of media like YYYY/MM/“YYYY-MM-DD_HHMMss.filetype”

However I now want to add my other type of media that are taken outside of my phone and are more occasion driven such as: DSLR photo sessions I had back when I was amateur photographer, my engagement photos, wedding etc. For these types of media I prefer to have them all enclosed in an album.

I’m curious how should I create albums but still maintain an organization in my originals folder? Does creating an album from the webUI create a folder in my originals folder? Can I directly add albums in my originals folder song side my camera roll photos above? How do nested folders appear in Photoprism?

**Comment by graciousgrey** [comment, marked as answer]
You folder structure from the originals directory is displayed in the originals section https://demo.photoprism.org/library/files and the folder section https://demo.photoprism.org/folders. If you create albums via the WebUI, your files will not be moved within the originals directory. That wouldn't make sense as one file could be part of multiple albums. If you add a folder e.g. "wedding" to your originals directory and start index --> the wedding folder would appear in the folders and originals section. You might create an album from a folder see https://docs.photoprism.org/user-guide/organize/folders/#create-album-from-folder.
